Makefile

```diff
@@ -33,9 +33,12 @@ test-online:
 
 checkstyle: test-shellcheck test-yaml
 
+shfmt:
+	shfmt -w . $$(file --mime-type test/*.t | sed -n 's/^\(.*\):.*text\/x-shellscript.*$$/\1/p')
+
 test-shellcheck:
 	@which shfmt >/dev/null 2>&1 || echo "Command 'shfmt' not found, can not execute shell script formating checks"
-	shfmt -d .
+	shfmt -d . $$(file --mime-type test/*.t | sed -n 's/^\(.*\):.*text\/x-shellscript.*$$/\1/p')
```

Comment:
So is this now *only* looking at shell scripts under test/ and not any more other files?

Reply:
It's looking at `test/*.t` *additionally* to `.`